content/docs/introduction/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+title: FAQ
+sort_rank: 5
+---
+
+# Frequently Asked Questions
+
+## General
+
+### What is Prometheus?
+Prometheus is an open-source systems monitoring and alerting toolkit,
+with an active ecosystem. See the [overview](../overview).
+
+### How does Prometheus compare against \[other monitoring system\]?
+
+See the [comparison](../comparison) page.
+
+### What dependencies does Prometheus have?
+
+The main Prometheus server runs standalone and has no external dependencies.
+
+### Can Prometheus be made highly available?
+
+Yes, run identical Prometheus servers on two or more separate machines.
+Identical alerts will be deduplicated by the [Alertmanager](https://github.com/prometheus/alertmanager).
+
+The Alertmanager cannot currently be made highly available, but this is a goal.
+
+### What language is Prometheus written in?
+
+Most Prometheus components are written in Go. Some are also written in Java and Ruby.
+
+### Who wrote Prometheus?
+
+Prometheus was initially started privately by
+[Matt Proud](https://github.com/matttproud) and
+[Julius Volz](https://github.com/juliusv). The majority of its
+development has been sponsored by [SoundCloud](https://soundcloud.com).
+
+Other companies making active contributions include [Boxever](www.boxever.com)
+and [Docker](https://www.docker.com). A full list can be found in the
+[CONTRIBUTORS](https://github.com/prometheus/prometheus/blob/master/CONTRIBUTORS.md)
```

Comment:
Does it make sense to maintain that file? It will likely be out of sync regularly,and both git as well as github provide commiter lists.

What is the difference between a contributor and a committer, how do we intend to keep the file up-to-date otherwise and who is responsible for that?